source_http.go

```diff
@@ -43,13 +50,27 @@ func (s *HttpImageSource) fetchImage(url *url.URL) ([]byte, error) {
 		return nil, fmt.Errorf("Error downloading image: (status=%d) (url=%s)", res.StatusCode, req.URL.String())
 	}
 
+	// Read the body
 	buf, err := ioutil.ReadAll(res.Body)
 	if err != nil {
 		return nil, fmt.Errorf("Unable to create image from response body: %s (url=%s)", req.URL.String(), err)
 	}
 	return buf, nil
 }
 
+func (s *HttpImageSource) setAuthorizationHeader(req *http.Request, ireq *http.Request) {
+	auth := s.Config.Authorization
```

Comment:
In that case, it would be a good idea to mention in the docs that once set from command line imaginary will ignore X-Forward-Authorization header